autogpt/commands/command.py

```diff
@@ -24,19 +24,23 @@ def __init__(
         description: str,
         method: Callable[..., Any],
         signature: str = "",
-        enabled: bool = True,
+        enabled: bool | Callable[[Config], bool] = True,
         disabled_reason: Optional[str] = None,
     ):
         self.name = name
         self.description = description
         self.method = method
-        self.signature = signature if signature else str(inspect.signature(self.method))
```

Comment:
this breaks everything: if we have config as default, then when someone doesn't put a signature (like list_agents) we're actually putting the config in the argument.
The fix is simple: the signature is now mandatory.